Week10/BlockbusterRecycler/app/src/main/java/com/e/blockbusterrecycler/ui/LoginActivity.kt

```diff
@@ -0,0 +1,71 @@
+package com.e.blockbusterrecycler.ui
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.view.View
+import android.view.inputmethod.InputMethodManager
+import androidx.appcompat.app.AppCompatActivity
+import com.e.blockbusterrecycler.R
+import kotlinx.android.synthetic.main.activity_login.*
+
+class LoginActivity : AppCompatActivity() {
+
+
+override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    setContentView(R.layout.activity_login)
+
+    val sharedPref = getSharedPreferences(getString(R.string.saveData), Context.MODE_PRIVATE)
```

Comment:
I believe this shared preferences object could've been injected, although Filip and Luka indicated that the injection should be the very last dependency, so the viewModel class(es) should be injected into their respective activities/fragments only, even though that deviates from what the video series does.